demo/src/assets/styles/index.css

```diff
@@ -19,7 +19,11 @@
   }
 
   a {
-    @apply hover:underline underline-offset-4;
+    @apply underline underline-offset-4 hover:text-slate-400;
+  }
+
+  a[target="_blank"]::after {
+    content: " 🔗";
```

Comment:
We will use "sqash and merge", which results in a single commit on the main branch. This pull request contains both CI improvements and code adjustments for the demo site. You might want to separate these...

Reply:
No need, I think we're fine since it's all related to the repo initial setup.